tests/observability/runbook_url/conftest.py

```
@@ -17,51 +17,89 @@ def cnv_prometheus_rules_names(hco_namespace):
+                    sleep=TIMEOUT_10SEC,
+                    func=session.head,
+                    exceptions_dict={
+                        requests.exceptions.ConnectionError: [],
+                        requests.exceptions.Timeout: [],
+                    },
+                    url=raw_url,
+                    timeout=TIMEOUT_10SEC,
+                ):
+                    if sample.status_code == requests.codes.ok:
+                        available_urls.add(runbook_url)
+                        LOGGER.info(f"Runbook URL reachable: {raw_url}")
```

Ruff (0.15.18)

[warning] Logging statement uses f-string

(G004)